TLM/TLM/State/ConfigData/Main.cs

```diff
@@ -10,13 +10,15 @@ public class Main {
 		/// Main menu button position
 		/// </summary>
 		public int MainMenuButtonX = 464;
+
```

Comment:
Odd positioning for a linebreak. These two variables are related to each other, why put a blank line? They should be next to each other.

Reply:
General rule "if it has a docstring, then it stands apart from others". This is not an automatically enforced rule, I've done it manually, it draws attention to other stuff which might be documented. I'll just remove this newline for now.